### PR TITLE
[CI:DOCS] DISTRO_PACKAGE.md: List the packaging changes for v4

### PR DESCRIPTION
**Pull request by lsm5**
Signed-off-by: Lokesh Mandvekar <lsm5@fedoraproject.org>

### CODE REVIEW

**Comment by lsm5** [comment]
once it's done here, it can go to the podman.io repo and other places.

**Comment by lsm5** [comment]
@containers/podman-maintainers @ashcrow @rhatdan @TomSweeneyRedHat @vrothberg @baude @mheon PTAL

File: DISTRO_PACKAGE.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
+# Podman Packaging
+
+This document is currently written with Fedora as a reference, intended for use
+by packagers of other distros as well as package users.
+
+## Fedora Users
+Podman v4 is available as an officlal Fedora package on Fedora 36 and rawhide.
+The latest version of Podman brings with it a new container stack called
```
Comment (vrothberg) [inline DISTRO_PACKAGE.md:8]:
```suggestion
This version of Podman brings with it a new container stack called
```

File: DISTRO_PACKAGE.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+Netavark which serves as a replacement for CNI plugins
+(containernetworking-plugins on Fedora), as well as Aardvark-dns, the
+authoritative DNS server for container records.
+
+Both Netavark and Aardvark-dns are available as official Fedora packages on
+Fedora 35 and newer versions and form the default network stack for new
+installations of Podman.
```
Comment (vrothberg) [inline DISTRO_PACKAGE.md:15]:
```suggestion
installations of Podman 4.0.
```

File: DISTRO_PACKAGE.md [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+```console
+$ sudo dnf copr enable rhcontainerbot/podman4
+
+# install or update per your needs
+$ sudo dnf install podman
+```
+
+After installation, if you would like to migrate all your containers to use
+Netavark, you will need to set `network_backend = "netavark"` under
+the `[network]` section in your containers.conf (typically located at:
```
Comment (vrothberg) [inline DISTRO_PACKAGE.md:44]:
```suggestion
the `[network]` section in your containers.conf, typically located at:
```

File: DISTRO_PACKAGE.md [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+`/usr/share/containers/containers.conf`
+
+If you would like to test the latest unreleased upstream code, try the
```
Comment (vrothberg) [inline DISTRO_PACKAGE.md:47]:
```suggestion
### Testing the latest development version`

If you would like to test the latest unreleased upstream code, try the
```

**Comment by rhatdan** [comment]
LGTM with @vrothberg change.

File: DISTRO_PACKAGE.md [hunk 5]
```diff
@@ -0,0 +1,101 @@
+# Podman Packaging
+
+This document is currently written with Fedora as a reference, intended for use
+by packagers of other distros as well as package users.
+
+## Fedora Users
+Podman v4 is available as an officlal Fedora package on Fedora 36 and rawhide.
```
Comment (ashley-cui) [inline DISTRO_PACKAGE.md:7]:
```suggestion
Podman v4 is available as an official Fedora package on Fedora 36 and rawhide.
```

**Pull request merged**